### PR TITLE
v2 manifest support

### PR DESCRIPTION
**Pull request by davidejones**
### What does this PR do?

This PR adds some support for v2 manifests by adapting the new key/values pairs to what the existing (v1) code expects. 
There are currently no v2 manifests live so you shouldn't see any difference on the preview. 
This code is tested locally as working against v2 draft prs. 

### Motivation

marketplace discussions 

### Preview

Shouldn't see anything different than live here
https://docs-staging.datadoghq.com/david.jones/support-manifestv2/integrations/

### Additional Notes

This is the short term solution to provide support now while wrapping up the longer term ideas.

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by bgdeutsch — APPROVED** [review]

**Review by kayayarai — APPROVED** [review]
Approving, but I'll let you merge when you're around, just in case :)

**Pull request merged**
